lm_eval/cnn_dailymail.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Move this file into `lm_eval/tasks` (it is currently at the project root level)

Reply:
@jon-tow  whoops.